### DISCUSSION TITLE
Add a top bar with cell actions

### DISCUSSION THREAD

**Discussion by zerefwayne** [fonsp/Pluto.jl#1064] [Pluto development]
### What is that?

A sticky top bar that contains buttons for cell actions like copy, delete, run, stop, duplicate, etc. The main motivation comes from the problem I had in figuring out shortcuts to manipulate the cells.

All the major editing software like Google Docs, Google Sheets, Jupyter Notebook have buttons to do such tasks along with the shortcuts.

### How will it look?

*Inspired by Jupyter*

![image](https://user-images.githubusercontent.com/28949397/113839880-ba8ef700-97ad-11eb-9cdf-ab843afca5a0.png)

### What's the use?

Newcomers to Pluto might have some problems in getting used to the shortcuts but that shouldn't stop them from doing cell manipulations like a pro. We can provide hints on hover on the buttons using which they can get used to them.

**Comment by fonsp** [comment]
Instead of a feature request, can you rephrase your comment as a _statement of a problem_? We already have these buttons, and newcomers don't need to know shortcuts to use Pluto, right?
